modules/main.js

```diff
@@ -0,0 +1,40 @@
+import Books from './constructor.js';
+
+/* eslint-disable */
+import { link1, link2, link3 } from './pages.js';
```

Comment:
**[OPTIONAL]** Kindly, remove codes that are not in use to keep your codebase clean. Also, just as a suggestion when styling the `List` page try to handle to display `title` and `author` in more than one line like this 👇 

<img width="1669" alt="image" src="https://user-images.githubusercontent.com/34578631/172247741-818d26a5-b19e-4656-be42-16669f04ed76.png">